packages/hint/docs/user-guide/troubleshoot/summary.md

```diff
@@ -11,6 +11,17 @@ This section contains the most commons issues reported by users with
 potential fixes. If you run into something that is not documented
 here please [open an issue in the hint repo][new issue].
 
+## Windows Subsystem Linux
+
+Due to the project using binary packages (`canvas` and `iltorb`) you can find some problems related to building those packages on WSL:
+
+> Info looking for cached prebuild @ /home/mahome/.npm/_prebuilds/47fbee-iltorb-v2.4.3-node-v59-linux-x64.tar.gz
+> WARN install No prebuilt binaries found (target=9.10.0 runtime=node arch=x64 libc= platform=linux)
+> gyp ERR! build error 
+> gyp ERR! stack Error: not found: make
+
+This is solved by installing the prerequisites of those packages [iltorb]. **However, due to the hard dependency on Puppeter, we do not recommend using WSL to run webhint.**
```

Comment:
I would complement this sentence saying: 

but if you want to run webhint using WSL you need to create your own config file with this minimun content:
```
{ "extends": ["web-recommended"], "connector": "jsdom" }
```
And then we need to pass the configuration to hint:
```
npx hint -c ./path/to/.hintrc https://example.com
```

Examples taken from @antross in teams hehehehe.

Reply:
Also can you add a return around the 80 character mark? Otherwise the linter will fail.

Thanks 🙏

Reply:
Should we also add documentation for WSL2? It's in insider's only for now but that's what I've been using and we have already some in [`connector-puppeteer`](https://webhint.io/docs/user-guide/connectors/connector-puppeteer/#wsl-support)

Reply:
@molant I think we should add it.